### ISSUE TITLE
Error in if (!out$check_pass)

### ISSUE THREAD

**Issue by joelnitta** [ropensci-review-tools/pkgcheck#165]
When [running the GH action](https://github.com/joelnitta/dwctaxon/actions/runs/4141768436/jobs/7161692566), I get this warning and error:

```
  Warning: The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
  Error in if (!out$check_pass) { : missing value where TRUE/FALSE needed
  Calls: checks_to_markdown ... FUN -> summarise_check -> do.call -> output_pkgchk_ci
  Execution halted
```

I don't encounter this running `pkgcheck()` locally.
